src/program/community-booth.njk

```diff
@@ -0,0 +1,132 @@
+        </div>
+    </div>
+    <div class="lg:w-[30%] w-[80%] lg:mt-30 flex items-center justify-center mx-auto">
+        <img src="{{ env.baseUrl }}img/assets/team.svg" alt="Community Booths" class="w-[80%] transition-transform duration-100" style="animation: floating 2s ease-in-out infinite;">
+    </div>
+</div>
+
+<div class="main-container pt-16 md:pt-12 bg-lavender w-full h-auto px-[6%]">
+    <div class="text-md lg:text-xl pt-12 text-black lg:pr-10 text-justify font-light">
+        <div class="mb-12">
+            <div class="space-y-6 mb-8">
+                <div>
+                    <div class="black-han-sans-regular font-normal text-3xl md:text-5xl md:text-center text-[#000000] text-center lg:text-left"> 
+                        Guidelines
+                    </div>
+                    <div class="pt-4 text-md">
+                        <ul class="list-none pl-12 space-y-3">
+                            <li class="flex">
+                                <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon" class="w-4 h-4 mr-2 mt-1">
+                                <span>Make sure you have tickets for the conference.</span>
+                            </li>
+                            <li class="flex">
+                                <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon" class="w-4 h-4 mr-2 mt-1">
+                                <span>Make sure at least one person is always available at the booth during breaks.</span>
+                            </li>
+                            <li class="flex">
+                                <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon" class="w-4 h-4 mr-2 mt-1">
+                                <span>Don't use the community booth to conduct any monetary transactions.</span>
+                            </li>
+                            <li class="flex">
+                                <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon" class="w-4 h-4 mr-2 mt-1">
+                                <span>Follow the <a href="{{ env.baseUrl }}coc/policy/">PyCon India Code of Conduct</a>.</span>
+                            </li>
+                            <li class="flex">
+                                <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon" class="w-4 h-4 mr-2 mt-1">
+                                <span> <span class="font-bold">Note for commercial entities:</span> We’d love for you to consider <a href="{{ env.baseUrl }}sponsors/">Sponsoring PyCon</a> instead of applying for a community booth.</span>
```

Comment:
```suggestion
                                <span> <span class="font-bold">Note for commercial entities:</span> We’d love for you to consider <a href="{{ env.baseUrl }}sponsors/">sponsoring PyCon India</a> instead of applying for a community booth.</span>
```